packages/apps-config/src/endpoints/development.ts

```diff
@@ -13,6 +15,8 @@ interface EnvWindow {
   }
 }
 
+const nodes = [{ link: 'wss://testnet.node.sydney.ggxchain.io', name: 'SYDNEY' }, { link: 'wss://testnet.node.brooklyn.ggxchain.io', name: 'BROOKLYN' }];
```

Comment:
It would be nice to move it later to some config, rather to have it hardcoded. However, it's just a TODO, not critical now.